.github/workflows/filecopymethod-test.yml

```diff
@@ -0,0 +1,88 @@
+# Some platform-specific file copy syscalls (e.g. creating reflinks) are only
+# supported on some platforms, and only with specific filesystems. These
+# syscalls are used by different FileCopyMethod implementations.
+#
+# This workflow sets up the conditions needed for those syscalls to work,
+# and then runs the tests with the different FileCopyMethods.
+
+name: FileCopyMethod
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.environment.runner }}
+    strategy:
+      matrix:
+        environment:
+         - runner: macos-latest
+           filesystem: APFS
+           copymethod: GetBytes
+         - runner: ubuntu-latest
+           filesystem: btrfs
+           copymethod: GetBytes
```

Comment:
might be typo of `CopyByte`, not `GetBytes`

Reply:
Same